airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/postgres/test_nested_streams/models/generated/airbyte_incremental/test_normalization/nested_stream_with_c___long_names_partition.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+    indexes = [{'columns':['_airbyte_emitted_at'],'type':'hash'}],
+    unique_key = '_airbyte_ab_id',
+    schema = "test_normalization",
+    tags = [ "nested" ]
+) }}
```

Comment:
Un-nesting an object type column from the parent stream can re-use the same `_airbyte_ab_id` column as unique_key